packages/relayer/core/src/config.rs

```diff
@@ -42,20 +45,45 @@ pub struct ServerConfig {
     pub address: String,
     /// The port to bind the server to.
     pub port: u16,
-    /// The log level for the server.
-    #[serde(default)]
-    pub log_level: String,
 }
 
 /// Returns true, used as a default value for boolean fields.
 const fn default_true() -> bool {
     true
 }
 
-impl ServerConfig {
-    /// Returns the log level for the server.
+/// Configuration for tracing and observability.
+#[derive(Clone, Debug, serde::Deserialize, serde::Serialize)]
+#[serde(default)]
+pub struct TracingConfig {
+    /// The log level to use.
+    pub level: String,
+    /// Whether to include file paths in logs.
+    pub with_file: bool,
+    /// Whether to use OpenTelemetry for distributed tracing.
+    pub use_otel: bool,
+    /// The service name to use for OpenTelemetry.
+    pub service_name: String,
+    /// The OpenTelemetry collector endpoint.
+    pub otel_endpoint: Option<String>,
+}
```

Comment:
Is this necessary? Don't we get most of this for free just by using Prometheus?

Reply:
It seems like most of these fields are either only nice-to-have or not necessary at all. I don't want to add a bunch of config fields unless necessary